lib/modules/manager/gleam/extract.ts

```diff
@@ -0,0 +1,48 @@
+import { HexDatasource } from '../../datasource/hex';
+import type { PackageDependency, PackageFileContent } from '../types';
+import { GleamToml } from './schema';
+
+function toPackageDep({
+  name,
+  version,
+  dev,
+}: {
+  name: string;
+  version: string;
+  dev?: boolean;
+}): PackageDependency {
+  return {
+    depName: name,
+    depType: dev ? 'devDependencies' : 'dependencies',
```

Comment:
we usually use the section key for this. that will make it more easily extendable in future